lib/class-experimental-wp-widget-blocks-manager.php

```diff
@@ -330,6 +330,12 @@ public static function convert_blocks_to_widget( $blocks ) {
 				'blocks' => $blocks,
 			)
 		);
+		wp_register_widget_control(
+			$widget_id,
+			__( 'Blocks Area', 'gutenberg' ),
+			'echo',
```

Comment:
This still triggers an error:
```
call_user_func_array() expects parameter 1 to be a valid callback, function 'echo' not found or invalid function name
````
It seems echo is not considered a callback function, but using a function that prints something would be dangerous anyway as printing something during REST request may make the requests invalid.
It seems our best bet is to create a simple noop function ourselves and pass its name here
.

Reply:
Will do that 👍